src/Routes/ImageManager/Images.js

```diff
@@ -22,14 +23,20 @@ const UpdateImageWizard = React.lazy(() =>
   )
 );
 
-const Images = () => {
-  const history = useHistory();
-  const { pathname, search } = useLocation();
+const Images = ({ historyProp, locationProp, navigateProp }) => {
+  const history = historyProp
+    ? historyProp()
+    : navigateProp
+    ? useNavigate()
+    : useHistory();
```

Comment:
I don't think we should use navigate in this var. We should return null if history isn't available.
```suggestion
 const history = historyProp
    ? historyProp()
    : useHistory
    ? useHistory()
    : null;
```